spog/ui/src/pages/search.rs

```diff
@@ -155,6 +155,48 @@ pub fn search(props: &SearchProperties) -> Html {
         },
     );
 
+    let onsort_advisory = {
+        let search_params_state = advisory.search_params.clone();
+        use_callback(
+            move |sort_by: (String, bool), search_params| {
+                if let SearchMode::Simple(simple) = search_params {
+                    let mut simple = simple.clone();
+                    simple.set_sort_by(sort_by);
+                    search_params_state.set(SearchMode::Simple(simple));
+                };
+            },
+            (*advisory.search_params).clone(),
```

Comment:
I think you should be able to put `advisory.search_param.clone()` here, and then use `if let … = **search_params {}` inside the callback.

Reply:
I've tried to do it but the compiler complains about `SearchMode` not implementing the `Copy` trait.

Reply:
Ah right … it's `if let … = &**search_params {}` I guess.

Reply:
@ctron wuau, that trick `&**` really worked. Thanks Jens!